database/src/main/postgres/runs/V1.5.12___update_existing_additional_data.sql

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE OR REPLACE FUNCTION runs._update_existing_additional_data(
+    IN  i_fk_partitioning     BIGINT,
+    IN  i_additional_data     HSTORE,
+    IN  i_by_user             TEXT,
+    OUT ad_backup_performed   BOOLEAN
```

Comment:
actually since this is a private / internal function, all I need really is the boolean value whether the update happened or no - more complicated, i.e. status and status_text, are probably unnecessary complication here in this case since    there are no custom 'scenarios' (or 'paths') where the code would fail and we would need to base some logic on top of that